tests/test_dto.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Test module for testing DTOs"""
+import pytest
+
+from covid_microservice.core.models import Dto, NewSampleSubmission, PcrTest
+from covid_microservice.core.utils import make_sample_id
+
+
+def test_pcrtest_is_dto():
+    """Make sure PcrTest is detected as a subclass of Dto"""
+    pcrtest = PcrTest(
+        sample_id=make_sample_id(),
+        patient_pseudonym="Alfredo Barnelli",
+        submitter_email="test@test.com",
+        collection_date="2022-08-21T11:18",
+        access_token="1234567812345678",
+    )
+    assert issubclass(PcrTest, Dto)
+    assert isinstance(pcrtest, Dto)
```

Comment:
Generally, we should try to check the behavior rather than implementation. Moreover, you don't have to test trivial things. So I think these tests are not required.